apps/backend/src/libs/modules/token/libs/types/token.type.ts

```diff
@@ -0,0 +1,12 @@
+import { type JWTPayload } from "jose";
```

Comment:
each type should be in a separate file. or write inline function signatures (best choice)

Reply:
I just saw that in the logger it's done like this, so I thought that this would be the best approach. I'll change it, thanks.

Reply:
@what1s1ove I think it's ok here, it is not a common type, and that's the only place it is used